tool_commands/raster_checker/test_raster_checker.py

```diff
@@ -27,30 +34,28 @@ def setUp(self):
         self.metadata = MetaData(bind=engine)
         self.datamodel = DataModelSource(self.metadata)
         self.rc_entrees = RasterCheckerEntrees(self.datamodel, session)
-        self.all_raster_ref_expect = [
+
+    def test_if_testdata_exists(self):
+        """ we need 1 sqlite and 3 rasters """
+        self.assertTrue(os.path.isfile(self.test_sqlite_path))
+        for tif in ["test1.tif", "test2.tif", "test3.tif"]:
+            self.assertTrue(os.path.isfile(os.path.join(TEST_DATA_DIR, "rasters", tif)))
```

Comment:
Ik ben trouwens (ook in threeditoolbox, kijk maar in `dependencies.py`) op python 3's pathlib aan het overstappen.
Is dat hier ook een idee?